config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```
@@ -214,6 +214,40 @@ presubmits:
       testgrid-tab-name: pull-kubernetes-e2e-aks-engine-azure-file
       testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
       testgrid-num-columns-recent: '30'
+  - name: pull-kubernetes-e2e-capz-conformance
+    decorate: true
+    always_run: false
+    optional: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred: "true"
```

here you need 

```suggestion
            preset-azure-cred-only: "true"
           preset-azure-anonymous-pull: "true"
  ```
  
  to be able to use the capzci registry

---

Good catch

---

did you push an update? I don't see it fixed

---

pushed